misk-aws/src/main/kotlin/misk/cloud/aws/AwsEnvironmentModule.kt

```diff
@@ -3,7 +3,7 @@ package misk.cloud.aws
 import com.google.inject.Provides
 import misk.environment.EnvVarLoader
 import misk.environment.ForEnvVars
-import misk.inject.KAbstractModule
+import misk.guice.KAbstractModule
```

Comment:
I like the new module, but renaming the package seems like needless churn. Is this so we can have both concurrently?

Reply:
changed back